src/components/ChatLog.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import TimeStamp from './TimeStamp';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+    return (
+        <ul>
+            {props.entries.map((entry) => (
+                <ChatEntry
+                    key={entry.id}
+                    id={entry.id}
+                    sender={entry.sender}
+                    body={entry.body}
+                    timeStamp={entry.time}
```

Comment:
I see `entry.time` is being passed, does this key exist in the message data in the .json file?